translate/packages/translate/translate/index.js

```diff
@@ -54,7 +54,14 @@ async function _command(params, commandText, secrets = {}) {
   const axios = require('axios');
   const sourceLang = 'auto';
   let translatedText = '';
-  let {language: targetLang = 'en', text: sourceText, __client} = params;
+
+  let parts = commandText.split(" ");
```

Comment:
If the parameters stay, there's a simpler way to do this because `language` is part of the `params` as `params.language` and the text is `params.varArg`. But as noted above, I like your implementation here in making the target language entirely optional. 

Our command parser does not support an optional parameter when using a vararg which would've been another option.